src/components/custom-attributes-control/index.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import { i18n } from 'stackable'
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Fragment, useState, useEffect, render, unmountComponentAtNode,
+} from '@wordpress/element'
+import { __ } from '@wordpress/i18n'
+import { TextControl } from '@wordpress/components'
+
+const CustomAttributesControl = props => {
+	const [ customAttributes, setCustomAttributes ] = useState(
+		Array.isArray( props.value ) ?
+			props.value.map( attribute => {
+				const [ key, _value ] = attribute
+				const value = `"${ _value }"`
+				return [ key, value ].join( '=' )
+			} ).join( ' ' ) :
+			''
+	)
```

Comment:
Might need a `useEffect` here to adjust things when the `prop.value` changes externally